### ISSUE TITLE
Suggestion: ensure SSH key and GH are connected

### ISSUE THREAD

**Issue by barrrricade** [lewagon/setup#359]
After SSH has been created
`ssh -T git@github.com`

[Source](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/testing-your-ssh-connection)

**Comment by dmilon** [comment]
Hi @barrrricade :wave: Thank you for your suggestion :pray: 

At the end of the [SSH section of the setup](https://github.com/lewagon/setup/blob/master/windows.md#giving-your-public-key-to-github), we ask students to check that `✓ Public key added to your account` has been displayed. If so, it means that their SSH key and GH CLI are connected. 

Did you encounter a case where they were not?

**Comment by barrrricade** [comment]
Hi @dmilon I've finished LW around a year ago (573), I find that the LW setup docs is very good.
However I don't need a lot of the features such as rails and node. (python/ C++)

Git and CLI are still needed but I have to change a lot of the parameters.
I have to use GH and Gitlab so I also created a config file to have multiple SSH keys.
`✓ Public key added to your account` shows up but `ssh -T git@github.com` can verify the specific SSH key used.

Also I use - `ssh-add -t 3h ~/.ssh/id_rsa_github`  <- or whatever the name of the file. 
to disable SSH for 3 hours - instead of always need a PW or having PW completely disabled
really helpful for final projects

Cheers
